samples/helloworld.py

```diff
@@ -1,27 +1,67 @@
 import logging
+
+from goonj import core
+from goonj.conf.constants import Sev
+from goonj.core import get_smart_alert
+
 logger = logging.getLogger('myapp')
 hdlr = logging.FileHandler('/Users/sohitkumar/goonj.log')
 formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
 hdlr.setFormatter(formatter)
 logger.addHandler(hdlr)
 logger.setLevel(logging.INFO)
 
-from goonj import core
-from goonj.conf.constants import Sev
-from goonj.core import get_smart_alert
 
-#logger = logging.getLogger(__name__)
+# logger = logging.getLogger(__name__)
 
 
 def run_sample():
     # initilaize goonj
     core.Goonj(config_file_path='goonj.yaml')
-    x = get_smart_alert('source_name1', logger)
-    x.info("dsgfds", sev=Sev.HIGH, subject="subjecr", error_id="1", error=None,
-           tag_list=["tag1", "tag2"])
-    x.info("dsgfds", Sev.HIGH, "subjecr", "1", None,
-           ["tag1", "tag2"])
-    x.info("only message ")
+    source_name1_alert = get_smart_alert('source_name1', logger)
+    y = get_smart_alert('source_name1', logger)
+    source_name1_alert.info("with rule engine ", sev=Sev.HIGH, subject="subjecr", error_id="1234", error=None,
```

Comment:
Start all messages with Capital letters

Reply:
For each of the examples given below, the message should state what will happen with the alert so that it's easy for others to simply identify the configuration they should use for their use cases.

E.g. message should be "This alert will be raised when Exception is raised 4 times in a span of 1 min"